src/javascript/JContent/ContentRoute/MainActionBar/MainActionBar.jsx

```diff
@@ -5,18 +5,18 @@ import {DisplayAction} from '@jahia/ui-extender';
 import {useSelector} from 'react-redux';
 import styles from './MainActionBar.scss';
 
-const ButtonRenderer = getButtonRenderer();
 const ButtonRendererShortLabel = getButtonRenderer({labelStyle: 'short'});
 const ButtonRendererNoLabel = getButtonRenderer({labelStyle: 'none'});
 
 export const MainActionBar = () => {
-    const {path} = useSelector(state => ({path: state.jcontent.path}));
-
+    const {path, selection} = useSelector(state => ({path: state.jcontent.path, selection: state.jcontent.selection}));
+    let editPath = (selection && selection.length === 0) ? path : selection[0];
```

Comment:
Why taking the selection ? I can't see anything in description or mockup about that ?

Reply:
I need to have the selection to disable the buttons https://jira.jahia.org/browse/BACKLOG-12329 as it's described in the story